Lib/copy.py

```diff
@@ -197,6 +197,7 @@ def _deepcopy_atomic(x, memo):
 d[types.FunctionType] = _deepcopy_atomic
 d[weakref.ref] = _deepcopy_atomic
 d[property] = _deepcopy_atomic
+d[memoryview] = _deepcopy_atomic
```

Comment:
It is not correct. `memoryview` can refer to mutable object. Deep copy should create a copy of the underlying object, so mutating the copy will not affect the original object. If the underlying object is atomic, `deepcopy()` can return the same object.

The copy should keep the shape etc of the original.

Reply:
Understood!

Reply:
Updated :)